### DISCUSSION TITLE
Document confirming the purchase of a license

### DISCUSSION THREAD

**Discussion by magals** [datalust/seq-tickets#1813] [General]
hello, we recently purchased a license for a team from you, but we did not receive a file with the fact of paying for the license by mail. we need this document for accounting. 

is it possible to send it or how to request it?

**Comment by nblumhardt** [comment]
Hi @magals, thanks for reaching out. If you can contact us by email at support@datalust.co, and let us know which organization this is for, we'll be able to send this for you.

Best regards,
Nick
